Sources/SwiftDocCUtilities/ArgumentParsing/Options/PreviewOptions.swift

```diff
@@ -17,6 +17,13 @@ import Foundation
 public struct PreviewOptions: ParsableArguments {
     public init() { }
 
+    @Option(
```

Comment:
Since this is public API, we'd want this to be documented. It's also a good place to document what the default value is.

Reply:
Done! Thanks.